todo/src/actions/index.js

```diff
@@ -0,0 +1,16 @@
+export const ADDTODO_TASK = 'ADD_TODO';
```

Comment:
To be more consistent and clear it is generally considered better aproach to set the const same to the value so it should be const `ADD_TODO = "ADD_TODO"`.